### PR TITLE
Update lrz for 320

### PR DESCRIPTION
**Pull request by fluidvanadium**

### CODE REVIEW

File: libtonode-tests/tests/concrete.rs [hunk 1]
```diff
@@ -1208,7 +1208,7 @@ mod slow {
         let first_send_to_transparent = 20_000;
         let summary_external_transparent = TransactionSummaryBuilder::new()
             .blockheight(BlockHeight::from_u32(7))
-            .status(ConfirmationStatus::Mempool(BlockHeight::from_u32(7)))
+            .status(ConfirmationStatus::Transmitted(BlockHeight::from_u32(7)))
```
Comment (zancas) [inline libtonode-tests/tests/concrete.rs]: This belongs in a different PR.

**Review by zancas — CHANGES_REQUESTED** [review]
It seems like two PRs are mixed together..  I am guessing the that LRZ update is atop another PR?

File: zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs [hunk 2]
```diff
@@ -289,60 +289,6 @@ impl InputSource for TransactionRecordsById {
     ) -> Result<Option<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
         unimplemented!()
     }
-    /// Returns a list of unspent transparent UTXOs that appear in the chain at heights up to and
```
Comment (AloeareV) [inline zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs:292]: I don't see these added back anywhere...does the new librustzcash make them no longer needed?

Comment (fluidvanadium) [reply zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs:292]: i am not sure these were ever needed

**Comment by zancas** [comment]
What's the state of this PR?  Is it superseded?

File: zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs (hunk 2 above)
Comment (AloeareV) [reply zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs:292]: Running against latest dev with the following diff, several tests fail. They're needed on latest dev. If they're not needed with new librustzcash, that's great! 
```diff
git diff
diff --git a/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs b/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
index f9abbb77..aa90003e 100644
--- a/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
+++ b/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
@@ -44,22 +44,4 @@ impl InputSource for TxMapAndMaybeTrees {
             .select_spendable_notes(account, target_value, sources, anchor_height, exclude)
             .map_err(TxMapAndMaybeTreesTraitError::InputSource)
     }
-
-    fn get_unspent_transparent_output(
-        &self,
-        _outpoint: &zcash_primitives::transaction::components::OutPoint,
-    ) -> Result<Option<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
-        unimplemented!()
-    }
-
-    fn get_unspent_transparent_outputs(
-        &self,
-        address: &zcash_primitives::legacy::TransparentAddress,
-        max_height: zcash_primitives::consensus::BlockHeight,
-        exclude: &[zcash_primitives::transaction::components::OutPoint],
-    ) -> Result<Vec<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
-        self.transaction_records_by_id
-            .get_unspent_transparent_outputs(address, max_height, exclude)
-            .map_err(TxMapAndMaybeTreesTraitError::InputSource)
-    }
 }
 ```

**Comment by fluidvanadium** [comment]
this PR is being pulled into smaller pieces

**Comment by zancas** [comment]
OK I will mention it in the tracking Issue, or should we close it?

**Pull request closed without merging**